classes/class-wc-rest-connect-address-normalization-controller.php

```diff
@@ -65,12 +71,24 @@ public function update_items( $request ) {
 		);
 		$response = $this->api_client->send_address_normalization_request( $body );
 
+		if ( is_wp_error( $response ) ) {
+			$error = new WP_Error(
+				$response->get_error_code(),
+				$response->get_error_message(),
+				array( 'message' => $response->get_error_message() )
+			);
+			$this->logger->log( $error, __CLASS__ );
+			return $error;
+		}
+
 		if ( isset( $response->error ) ) {
-			return new WP_Error(
+			$error = new WP_Error(
 				$response->error->code,
 				$response->error->message,
 				array( 'message' => $response->error->message )
 			);
```

Comment:
You could use [`WP_Error::add_data()`](https://developer.wordpress.org/reference/classes/wp_error/add_data/) here instead of creating a new `WP_Error`.